### ISSUE TITLE
Explicitly specifying language on fenced block doesn't work

### ISSUE THREAD

**Issue by harpocrates** [lightbend/paradox#454]
Putting an explicit language tag on a fenced code block does not work:

    ```mylang
    foo bar
    ```

It generates the following (which I've indented):

```html
<pre class="prettyprint" id="__code_0">
  <button class="md-clipboard" title="Copy to clipboard" data-clipboard-target="#__code_0 pre, #__code_0 code">
    <span class="md-clipboard__message"></span>
  </button>
  <code class="language-mylang">foo bar</code>
</pre>
```

However, `prettify` apparently expects the following instead (noticed the `language-mylang` class has moved):

```html
<pre class="prettyprint language-mylang" id="__code_0">
  <button class="md-clipboard" title="Copy to clipboard" data-clipboard-target="#__code_0 pre, #__code_0 code">
    <span class="md-clipboard__message"></span>
  </button>
  <code>foo bar</code>
</pre>
```

---

Workaround for other poor souls: make sure this gets run before `PR.prettyPrint()`:

```javascript
// Lift the `language-*` classnames from `code` elements up onto their `pre` parent elements
var codeElems = document.body.getElementsByTagName("code");
for (var i = 0; i < codeElems.length; ++i) {
   var codeElem = codeElems[i];
   var languageClassesToLift = [];
   codeElem.classList.forEach(function (className) {
     if (className.startsWith("language-")) languageClassesToLift.push(className);
   });
   if (codeElem.parentElement.classList.contains("prettyprint")) {
     DOMTokenList.prototype.add.apply(codeElem.parentElement.classList, languageClassesToLift);
   }
}
```

**Comment by harpocrates** [comment]
I think this is affecting the Akka docs too: notice how on https://doc.akka.io/docs/akka/current/typed/actors.html#akka-actors the `val` keywords often don't get colored properly (because the `language-scala` is not getting picked up). 

Before:
<img width="358" alt="Screen Shot 2020-10-25 at 3 41 17 PM" src="https://user-images.githubusercontent.com/10766081/97117225-a8763c00-16d8-11eb-8ffd-c13c6da9497b.png">

After applying the workaround I mentioned above:
<img width="379" alt="Screen Shot 2020-10-25 at 3 36 53 PM" src="https://user-images.githubusercontent.com/10766081/97117126-1f5f0500-16d8-11eb-8ed0-2d277b96ca56.png">

**Comment by ennru** [comment]
Thank you reporting. Scala seems to be very little affected by this.
There is even #450 suggesting to switch the library for syntax highlighting.

Either way, please suggest the necessary changes in a PR if you are in a position to do that.

**Comment by harpocrates** [comment]
> There is even #450 suggesting to switch the library for syntax highlighting.

Right, I was the one who opened that too 😄. I think I'll focus my efforts on switching to `highlight.js` instead of fixing this - the main reason for opening the bug is to make sure others running into this issue realize what is causing it (and know how to work around it).
